util/gvalid/gvalid_validator.go

```diff
@@ -124,6 +133,13 @@ func (v *Validator) Ci() *Validator {
 	return newValidator
 }
 
+// JsonTagAsAlias sets using the JSON tag value as an alias name.
```

Comment:
FieldBail  sets using the JSON tag value as alias when the alias name is not set in the validation rule.

Reply:
fixed